packages/nx-plugin/src/generators/app/files/template-angular-v16/src/wait-for.ts__template__

```diff
@@ -0,0 +1,19 @@
+import { firstValueFrom, isObservable, Observable } from 'rxjs';
```

Comment:
Should this be moved into the `@analogjs/trpc` package?

Reply:
Also, this file should not be generated if tRPC is not added

Reply:
I am fine with moving it to the `@analog/trpc` package. Or should we move it to `@analogjs/platform` since it might also be helpful for people using the http client

Reply:
moved wait for utility to trpc package